examples/hosts/app-integration/external-data/src/model/containerCode.ts

```diff
@@ -13,6 +13,9 @@ import { AppModel } from "./appModel";
 import { TaskListInstantiationFactory } from "./taskList";
 
 const taskListId = "task-list";
+const SignalType = {
+    ExternalDataChanged: "externalDataChange"
```

Comment:
Nit: this might be a good spot to document what this signal represents / how we use it.